src/Event/PreReadEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace ApiPlatform\Core\Event;
+
+use ApiPlatform\Core\DataProvider\CollectionDataProviderInterface;
+use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
+use ApiPlatform\Core\DataProvider\SubresourceDataProviderInterface;
+use ApiPlatform\Core\Identifier\IdentifierConverterInterface;
+use ApiPlatform\Core\Serializer\SerializerContextBuilderInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class PreReadEvent extends Event
+{
+    const NAME = 'api_platform.pre_read';
```

Comment:
You should store the names in a dedicated class, like https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/FormEvents.php